test.js

```diff
@@ -17,26 +16,26 @@ Object.keys(ansi).forEach(function (x) {
 	process.stdout.write(style + x + ansi.reset.open + ansi.reset.close + ' ');
 });
 
-it('should return ANSI escape codes', function () {
-	assert.equal(ansi.green.open, '\u001b[32m');
-	assert.equal(ansi.bgGreen.open, '\u001b[42m');
-	assert.equal(ansi.green.close, '\u001b[39m');
-	assert.equal(ansi.gray.open, ansi.grey.open);
+test('should return ANSI escape codes', t => {
+	t.is(ansi.green.open, '\u001b[32m');
+	t.is(ansi.bgGreen.open, '\u001b[42m');
+	t.is(ansi.green.close, '\u001b[39m');
+	t.is(ansi.gray.open, ansi.grey.open);
 });
 
-it('should group related codes into categories', function () {
-	assert.equal(ansi.colors.magenta, ansi.magenta);
-	assert.equal(ansi.bgColors.bgYellow, ansi.bgYellow);
-	assert.equal(ansi.modifiers.bold, ansi.bold);
+test('should group related codes into categories', t => {
+	t.is(ansi.colors.magenta, ansi.magenta);
+	t.is(ansi.bgColors.bgYellow, ansi.bgYellow);
+	t.is(ansi.modifiers.bold, ansi.bold);
 });
 
-it('groups should not be enumerable', function () {
-	assert.equal(Object.keys(ansi).indexOf('modifiers'), -1);
+test('groups should not be enumerable', t => {
+	t.is(Object.keys(ansi).indexOf('modifiers'), -1);
```

Comment:
IMHO too obscure, and all these kinds of checks will have much better debuggability when we add support for [power-assert](https://github.com/power-assert-js/power-assert).